### DISCUSSION TITLE
Default program to open images seems off

### DISCUSSION THREAD

**Discussion by lucrnz** [minios-linux/minios-live#12] [Ideas]
Hey! So far MiniOS seems great, I have it on my USB thumb-drive as a rescue O.S.

I would love to suggest a change in the default program to launch image files,  it seems like the program "PDF Arranger" is set as default.

The user experience to open a picture is double-click it on the file explorer and find an error about this PDF program not being able to see the image.

Can you folks change this to the XFCE image viewer?

As a note, I am using the "maximum" version.

Thank you!

**Comment by crim50n** [comment]
Thanks for the suggestion, we will try to make the appropriate changes in a future version of MiniOS.

**Comment by fabio8v** [comment]
Hi..just a simple questio..what's the root password?

**Reply by crim50n** [reply]
```
live:evil
root:toor
```
